dracclient/client.py

```diff
@@ -753,6 +753,57 @@ def list_raid_controllers(self):
         """
         return self._raid_mgmt.list_raid_controllers()
 
+    def list_raid_controller_settings(self, by_name=False):
```

Comment:
I don't think a "by_name" parameter here makes sense because there are several attributes that are per virtual disk, which means there could be multiple attributes with the same name, but different instance IDs for a single RAID controller.  Let's remove this parameter to keep users from shooting themselves in the foot.